### PR TITLE
 Set UID,fsGroup and Linux options to cassandra RC's

### PR DESCRIPTION
**Pull request by rubenvp8510**
This PR fixes BZ 1570583 about Cassandra permissions.

Also, revert the changes made to migrate hawkular-metrics to another namespace, because this migration introduces regression bugs (See BZ 1590748).

This PR fixes a bug on 3.10.

### CODE REVIEW

File: roles/openshift_metrics/tasks/install_cassandra.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@
     master: "{{ (item == '1')|string|lower }}"
     replica_count: "{{cassandra_replica_count.results[item|int - 1].stdout}}"
     node_selector: "{{openshift_metrics_cassandra_nodeselector | default('') }}"
+    fsgroup: "{{ openshift_metrics_namespace_fsgroup | default(false) }}"
```
Comment (liggitt) [inline roles/openshift_metrics/tasks/install_cassandra.yaml]: does the default mean it defaults to a literal `"false"`?

File: roles/openshift_metrics/tasks/pre_install.yaml [hunk 2]
```diff
@@ -19,3 +19,55 @@
     get secrets -o name
   register: metrics_secrets
   changed_when: false
+
+- name: Get namespace object
+  oc_obj:
+    state: list
+    kind: project
```
Comment (liggitt) [inline roles/openshift_metrics/tasks/pre_install.yaml]: prefer get namespace over list project

**Comment by rubenvp8510** [comment]
@liggitt I've already addressed all your comments, I think this is ready for another review

//cc @jsanda @ewolinetz

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by ewolinetz** [comment]
I think you need to rebase this PR. It seems to be reverting the changes made to move to another project that resolved a different BZ.
/hold

**Comment by liggitt** [comment]
> I think you need to rebase this PR. It seems to be reverting the changes made to move to another project that resolved a different BZ.
> /hold

that is intentional. the move revealed other components that depended on the original namespace. this is resolving the same bug in a different way.

**Comment by ewolinetz** [comment]
bot, retest this please

**Pull request merged**